### DISCUSSION TITLE
failed to download vulnerability DB

### DISCUSSION THREAD

**Discussion by Douglas235** [aquasecurity/trivy#4960] [Q&A] [answered]
### Question

Why does the DB initialization error occur？
![图片](https://github.com/aquasecurity/trivy/assets/66821741/0e312662-c9bc-4aad-802f-079a184fb50d)

### Target

None

### Scanner

None

### Output Format

None

### Mode

None

**Comment by nikpivkin** [comment, marked as answer]
Hi @Douglas235 !

Most likely you have a slow internet connection and the timeout [has been exceeded](https://aquasecurity.github.io/trivy/v0.44/docs/references/troubleshooting/). Try running Trivy with the `--timeout` flag and specify the timeout duration, for example `--timeout 30m`.

**Reply by Douglas235** [reply]
Thank you for your help
